tests/test_dataset.py

```diff
@@ -27,9 +27,19 @@
 import unittest
 
 import lsst.utils.tests
+import lsst.pex.exceptions as pexExcept
 from lsst.ap.verify.dataset import Dataset
 
 
+haveDataset = False
+try:
+    lsst.utils.getPackageDir("ap_verify_hits2015")
+    haveDataset = True
+except pexExcept.NotFoundError:
+    pass
+
+
+@unittest.skipIf(not haveDataset, "ap_verify_hits2015 not installed")
```

Comment:
The other option is to do this in `setUpClass()`, as below. I'm not sure it matters which you use, although if you do need that packageDir in the class (e.g., for butler things), using setUpClass makes that easy. You should change the `not installed` to `not setup`, since it's the latter that really matters.

```
@classmethod
def setUpClass(cls)
        try:
            cls.dataDir = lsst.utils.getPackageDir("ap_verify_hits2015")
        except lsst.pex.exceptions.NotFoundError:
            raise unittest.SkipTest("ap_verify_hits2015 not setup")
```

Reply:
In this case it's "not installed", because the `Dataset` class sets up a dataset package during initialization. A bit different from our standard use of EUPS, I know.